app/controllers/home_controller.rb

```diff
@@ -1,8 +1,13 @@
 class HomeController < ApplicationController
+
+  SLOTS_WINDOW_IN_DAYS = 3
```

Comment:
Como é um refactor simples, acredito que já vale fazermos pra eliminar o risco de esquecermos de alterar um dos locais. Pensei em colocarmos na classe `Appointment`:

###### app/models/appointment.rb
```ruby
class Appointment < ApplicationRecord
  SLOTS_WINDOWS_IN_DAYS = 3

  ...
end
```

E referenciarmos aqui e no `TimeSlotController`.

Reply:
Outra alternativa seria colocar essa informação em uma variável de ambiente

Reply:
Boa, @iago-silva. Concordo.

```suggestion
  ENV['SLOTS_WINDOW_IN_DAYS']&.to_i || 3
```